src/scripts/app.js

```diff
@@ -22,11 +22,22 @@ class App extends JSXComponent {
   }
 
   getBudget() {
-    chrome.storage.sync.get(defaultBudget, (data) => {
-      this.setState({
-        budget: data
+    if (!chrome.storage.hasOwnProperty("sync")) {
+      // Firefox has no sync right now...
```

Comment:
Here's the tracking bug for that feature, might be good in a comment? 

https://bugzilla.mozilla.org/show_bug.cgi?id=1220494